jenkinshelper.ksh

```diff
@@ -0,0 +1,139 @@
+#!/bin/ksh
+# 2021-04-07 Olaf Bohlen <olbohlen@eenfach.de>
+# instead of putting all this into the Jenkinsfile I decided to put the actual code in this script
+
+# global config
+HTTPCONF="/etc/apache2/2.4/conf.d/pkgdepotd.conf"
+
+# just run prepare once to initially set up the environment
```

Comment:
Seems this part needs to be run as root, makes sense to document as such

Reply:
Ok done, let me know where to explicitly document it if you have a good place to put it.